kotlin-native/backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/objcexport/ObjCExportNamer.kt

```diff
@@ -162,14 +173,6 @@ private class ObjCExportNamingHelper(
         return ObjCExportNamer.ClassOrProtocolName(swiftName = baseName, objCName = "$topLevelNamePrefix$baseName")
```

Comment:
Should we allow the annotation on files as well to control the name of the file class?